docs/02-get-started/04-trigger-workload-with-event.md

```diff
@@ -0,0 +1,225 @@
+---
+title: Trigger a workload with an event
+---
+
+We already know how to create and expose a workload ([Function](02-deploy-expose-function.md) and [microservice](03-deploy-expose-microservice.md)). 
+Now it's time to actually use them.
+We're going to trigger a workload with an event. 
+
+## Create a Function 
+
+To trigger our workload, we need something to trigger it with. 
+For this, we are going to create a new Function with a corresponding OAuth2Client and an APIRule. As this is a sample Function, we'll make it so that it both sends and receives the events. 
+
+<div tabs name="Deploy a Function" group="trigger-workload">
+  <details open>
+  <summary label="kubectl">
+  kubectl
+  </summary>
+
+Run:
+<!--TODO: 1) Replace `{LASTORDER_FUNCTION}` below with the actual Function's code. -->
+
+```bash
+cat <<EOF | kubectl apply -f -
+  {LASTORDER_FUNCTION}
+---
+  apiVersion: hydra.ory.sh/v1alpha1
+  kind: OAuth2Client
+  metadata:
+    name: lastorder
+  spec:
+    grantTypes:
+      - "client_credentials"
+    scope: "read write"
+    secretName: lastorder-oauth
+---
+  apiVersion: gateway.kyma-project.io/v1alpha1
+  kind: APIRule
+  metadata:
+    name: lastorder
+  spec:
+    gateway: kyma-gateway.kyma-system.svc.cluster.local
+    rules:
+      - path: /function
+        methods: ["GET", "POST"]
+        accessStrategies:
+          - handler: oauth2_introspection
+            config:
+              required_scope: ["read"]
+      - path: /.*
+        methods: ["GET", "POST"]
+        accessStrategies:
+          - handler: allow
+            config: {}
+    service:
+      host: lastorder.$CLUSTER_DOMAIN
+      name: lastorder
+      port: 80
+EOF
+```
+
+If the resources were created successfully, the command returns this message:
+
+```bash
+function.serverless.kyma-project.io/lastorder created
+oauth2client.hydra.ory.sh/lastorder created
+apirule.gateway.kyma-project.io/lastorder created
+```
+
+  </details>
+  <details>
+  <summary label="Kyma Dashboard">
+  Kyma Dashboard
+  </summary>
+
+1. From the left navigation, go to **Functions** and click to create a new Function.
+2. Name the Function `lastorder` and click **Create**.
+3. In the inline editor for the Function, modify its source replacing it with this code: 
+   <!--TODO: Replace `{LASTORDER_FUNCTION}` below with the actual Function's code. -->
+    ```js
```

Comment:
Maybe it was because of the comment I had there. 
Anyway, I added the proper source code and removed the comment and it's visible now. 
![image](https://user-images.githubusercontent.com/53601578/133631771-ba6d3f22-5704-4d50-aff8-d85689b55c77.png)

Reply:
Yay!!